docs/health/health_usage/usage.rst

```diff
@@ -13,7 +13,11 @@ Integrated
 ----------
 Running `pyATS Health Check` integrated with pyATS scripts is the way if a user wants to collect and monitor device status on a testbed through a pyATS job.
```

Comment:
it seems this uses a 'default' health file? Can you elaborate/explain how this default file works and where to find it if users want to change the default arguments for these.

Reply:
yes. correct. I think no need to explain the default health file. that's the concept to use simply. and the default thing is explained with `--health-checks` at beginning and also how to change behavior by some arguments as well.

Reply:
Reading the 'usage' doc, I think we should add a pointer to the default health.yaml file for the cpu, memory, logging and core operations.

Reply:
as per request, added via 8cf25c0